lib/runner/Configuration.js

```diff
@@ -85,6 +85,11 @@ module.exports = class Configuration {
             // Load the configuration file for skill-testing, if there is one
             const configExplorer = cosmiconfig("testing", { searchPlaces });
             const skillConfigResult = await configExplorer.search();
+            if (!skillConfigResult.filepath) {
+                // eslint-disable-next-line no-console
+                console.error("Unable to find the configuration file");
+                return;
```

Comment:
Probably we need to set default values for this.configurationJSON and this.configurationJSON.configurationPath before returing.

Reply:
on that point we have already searched 
  "./test/unit/testing.json",
                "./test/unit/skill-testing.json",
                "./test/e2e/testing.json",
                "./test/e2e/skill-testing.json",
                "./test/testing.json",
                "./test/skill-testing.json",
                "testing.json",
                "skill-testing.json",
So there's no default to add

Reply:
added default value